manifest/yaml.go

```diff
@@ -113,7 +113,29 @@ func (v *Resource) SetName(name string) error {
 }
 
 func (v Services) MarshalYAML() (interface{}, error) {
-	return nil, fmt.Errorf("unimplemented")
+	services := make(map[string]interface{})
+
+	// Loop over all of the services
+	for _, s := range v {
+		service := make(map[string]interface{})
+
+		// If there is a build key
```

Comment:
There is an `applyDefaults` on Manifest that will set up all of the default values correctly.